src/plugins/plot/inspector/PlotOptionsBrowse.vue

```diff
@@ -197,36 +197,40 @@ export default {
     },
     methods: {
         initConfiguration() {
-            this.label = this.config.yAxis.get('label');
-            this.autoscale = this.config.yAxis.get('autoscale');
-            this.logMode = this.config.yAxis.get('logMode');
-            this.autoscalePadding = this.config.yAxis.get('autoscalePadding');
-            const range = this.config.yAxis.get('range');
-            if (range) {
-                this.rangeMin = range.min;
-                this.rangeMax = range.max;
-            }
+            if (this.config) {
```

Comment:
I don't know a lot about our plots, do any of these options have defaults already or need them? @shefalijoshi any thoughts on that?

Reply:
I know plots depend a good bit on configs, so this is most likely a temp solution as well as the fix in the StackedPlotItem.vue. I'm looking into another ticket with interceptors that looks like set defaults that could be a solution? We would need to figure out what defaults to set it too though.